### DISCUSSION TITLE
language Model

### DISCUSSION THREAD

**Discussion by sibtain-raxa** [apache/incubator-mxnet#15372] [Q&A]
does Mxnet deepspeech model supports language model(kenlm) based decoding?

**Comment by vdantu** [comment]
@mxnet-label-bot add [question]

**Comment by lanking520** [comment]
@Soonhwan-Kwon

**Comment by Soonhwan-Kwon** [comment]
No it does not but we did implemented kenlm based decoding with similar project before so it is possible to do.

**Comment by sibtain-raxa** [comment]
@Soonhwan-Kwon which project?
